yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,512 @@
+
+<hint markdown = "block" title = "Answer!">
+If done correctly, your blocks should look like this: 
+
+![Answer!](../images/number_guessing_game/image30.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+(Don’t forget the spaces! Additional note: in order to make the game a little bit more intense, in our example we added a pause before the Intent gives the answer. Feel free to to use adjust the pause time based on your preference, you can find the pause function in the voice drawer)
+</hint>
+
+## Define the Intent Handler for *makeGuessIntent* (Part 3)
+
+We've now programmed the **makeGuessIntent** such that **if myGuessSlot** is greater than our **secret** value, we tell the user information about **myGuessSlot** and prompt the user to guess again. But what if **myGuessSlot** is smaller than our **secret** value? We need to define the **else if** and **then** statement to have Alexa know what to do in this situation.
+
+In order to define the **else if** statement, follow the same steps as before for defining the **if statement**, using the <span class = "math">equality</span> function block to compare **myGuessSlot** and our **secret** value - but this time change the greater than sign to a less than sign (<).
+
+Now let's define the second **then** statement. Follow the same steps as the first **then** statement, but instead of having Alexa **ask**: "(myGuessSlot) is too **big**. Guess again", we want Alexa to **ask**: "(myGuessSlot) is too **small**. Guess again". 
+
+Your blocks should look like this! 
+
+![Adding else if/then](../images/number_guessing_game/image20.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Finally, we want to define what happens if **myGuessSlot** is not bigger or less than **secret** (equal to the value of **secret**). To do so, we need to define the **else** statement). 
+
+## Define the Intent Handler for *makeGuessIntent* (Part 4)
+
+If our guess is correct and equal to secret, we no longer need to continue the conversation. Therefore, instead of using **ask**, we will use **say** instead. We want Alexa to **say**: "(myGuessSlot) is correct." To do this, we need to use the **say** block from the **Voice** drawer, the **myGuessSlot.value** block, and the <span class = "text">join</span> block from the **Text** drawer. Try it yourself! (Hint: How many parts are there in Alexa's responses? Do we need to add parts to the <span class = "text">join</span> block?)
```

Comment:
"equal to secret," formatting of secret missing.
(myGuessSlot)  formatting issue again